lib/ltutils.coffee

```diff
@@ -34,6 +35,9 @@ module.exports.get_tex_root = (editor) ->
     root = path.resolve(path.dirname(root), directives.root)
   return root
 
+  if project?.TEXroot?
```

Comment:
Almost there... I think 34-39 needs to look something like this:

```python
  if directives.root?
    return path.resolve(path.dirname(root), directives.root)

  if project?.TEXroot?
    return project.TEXroot

  return root
```

Otherwise, as is, now I don't think its possible to reach the `project.TEXroot` test.